ci-operator/step-registry/ipi/conf/azure/arcconformance/ipi-conf-azure-arcconformance-commands.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export PATH=$PATH:/tmp/bin
+mkdir /tmp/bin
+
+echo "Leased resource:"
+echo ${LEASED_RESOURCE}
+
+echo "Shared dir:"
+ls -la ${SHARED_DIR}
+
+echo "Cluster profile dir:"
+ls -la ${CLUSTER_PROFILE_DIR}
+
+echo "Install config:"
+cat ${SHARED_DIR}/install-config.yaml
+
+echo "Metadata:"
+cat ${SHARED_DIR}/metadata.json
+
+echo "$(date -u --rfc-3339=seconds) - Installing tools..."
+
+# install sonobuoy
+# TODO move to image
+curl -L https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.20.0/sonobuoy_0.20.0_linux_amd64.tar.gz | tar xvzf - -C /tmp/bin/ sonobuoy
+chmod ug+x /tmp/bin/sonobuoy
+sonobuoy version
+
+# install jq
+# TODO move to image
+curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+chmod ug+x /tmp/bin/jq
+jq
+
+# install yq
+# TODO move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 > /tmp/bin/yq 
+chmod ug+x /tmp/bin/yq
+yq --version
+
+# az should already be there
+az version
+
+echo "$(date -u --rfc-3339=seconds) - Collecting parameters..."
+
+# set the parameters we'll need as env vars
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+AZURE_AUTH_CLIENT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .tenantId)"
+AZURE_AUTH_SUBSCRIPTION_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .subscriptionId)"
+CLUSTER_NAME="$(yq r "${SHARED_DIR}/install-config.yaml" 'metadata.name')"
+RESOURCE_GROUP="${CLUSTER_NAME}-rg"
+REGION="$(yq r "${SHARED_DIR}/install-config.yaml" 'platform.azure.region')"
```

Comment:
You have a cluster up by this point.  Instead of relying on the install-time `install-config.yaml`, it's probably more reliable to extract the region from the in-cluster config.  Example of pulling AWS region from the cluster [here][1].  But I don't see a setting for Azure region [in the `Infrastructure` struct][2].  Is this test somehow the only Azure bit that needs someone to tell it what region to use?

[1]: https://github.com/openshift/release/blob/b09d2ab191ca9fab132a20a8371fa25f023a4105/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L92
[2]: https://github.com/openshift/api/blob/790e0a84e3e01dc3f6218e655b0e92b809ae7fd0/config/v1/types_infrastructure.go

Reply:
@wking I know this is in `cluster-infrastructure-02-config.yml` so yes, it should be there. I'll try. If all goes wrong this must be in `metadata.json` too which is a bit better than install-config I guess. ;-)

Reply:
Actually the region is not there, so now I'm taking it from `LEASED_RESOURCE`. It worked for the resource group and infra ID though.